auditlog/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================
+Audit Log - Track user operations
+=================================
+
+This module allows the administrator to log user operations performed on data
+models such as ``create``, ``read``, ``write`` and ``delete``.
+
+Usage
+=====
+
+Go to `Settings / Technical / Audit / Rules` to subscribe rules. A rule defines
```

Comment:
nice to have: for procedure, better use enumerated list (with `#. `).
nice-to-have: migrate to the new fragmented README file (see https://github.com/OCA/mis-builder/tree/10.0/mis_builder as an example)